app/controllers/pages_controller.rb

```diff
@@ -34,4 +40,15 @@ def simple
   def set_comments
     @comments = Comment.all.order("id DESC")
   end
+
+  def comments_json_string
+    render_to_string(template: "/comments/index.json.jbuilder",
+                     locals: { comments: Comment.all }, format: :json)
```

Comment:
This came from the Rails view code where we set the props directly on the on the component.